src/lib/components/navbar/Navbar.component.js

```diff
@@ -235,16 +216,14 @@ function NavBar({
       </NavbarMenu>
       {tabs.length ? (
         <NavbarTabs>
-          {tabs.map(({ link, title, selected, onClick }, index) =>
+          {tabs.map(({ link, title, selected, onClick }, index) => 
             link ? (
-              <TabLinkItem
-                selected={selected}
-                role="tab"
-                aria-selected={selected}
-                key={`navbar_tab_link_item_${index}`}
-              >
-                {link}
-              </TabLinkItem>
+              React.cloneElement(link,
+                {
+                  className: (selected ? "selected" : ""),
+                  "aria-selected": selected
```

Comment:
I think role="tab" and key props are missing

Reply:
Props `role` & `key` added